rfcs/2021-11-12-9811-vrl-vm.md

```diff
@@ -0,0 +1,164 @@
+# RFC 9811 - 2021-11-12 - VRL bytecode VM
+
+This RFC proposes implementing a bytecode VM. VRL will be compiled to the bytecode
+and executed by this VM, with the aim to significantly improve the performance of
```

Comment:
We should add a section to this RFC discussing (and proving) the performance gains we've seen from the early spike, as those gains are what warrants the effort needed to implement this RFC.